src/controllers/controller.bar.js

```diff
@@ -87,7 +87,13 @@ defaults._set('horizontalBar', {
 
 			label: function(item, data) {
 				var datasetLabel = data.datasets[item.datasetIndex].label || '';
-				return datasetLabel + ': ' + item.xLabel;
+
+                //float-bar support, if y arguments are array tooltip will show bottom and up values
+                if(!helpers.isArray(data.datasets[item.datasetIndex].data[item.index])) {
```

Comment:
I'd recommend creating a variable to store `data.datasets[item.datasetIndex].data[item.index]` since it is used three times. It will make the code smaller to download to the browser

Also, it might be a little easier to read the `if` statement if you reverse the conditions so that you don't need the `!`

Reply:
done!